launch.go

```diff
@@ -432,6 +432,7 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 		"SD_BUILD_URL":           apiURL + "builds/" + strconv.Itoa(buildID),
 		"SD_STORE_URL":           fmt.Sprintf("%s/%s/", storeURL, "v1"),
 		"SD_UI_URL":              fmt.Sprintf("%s/", uiURL),
+		"SD_UI_BUILD_URL":        uiURL + "/builds/" + strconv.Itoa(buildID),
```

Comment:
```suggestion
		"SD_UI_BUILD_URL":        uiURL + "builds/" + strconv.Itoa(buildID),
```

Reply:
@tkyi my original commit was what you suggested, it failed the unit test since if you check the line above `SD_UI_URL` is initialized by suffixing a `/` at end of uiURL

Reply:
as per line 434 SD_UI_URL already has a trailing slash 🤔